### ISSUE TITLE
Segoe Fluent Icons

### ISSUE THREAD

**Issue by forumrulez** [CKGrafico/Segoe-mdl2-assets-css#3]
Will the icons be updated to the new [Segoe Fluent Icons](https://docs.microsoft.com/en-us/windows/apps/design/style/segoe-fluent-icons-font)?

**Comment by CKGrafico** [comment]
mm not in the roadmap, really important to you? maybe more people can ask?

**Comment by forumrulez** [comment]
> mm not in the roadmap, really important to you? maybe more people can ask?

Yes, this is really important, I create applications on Electron, in which I use your icons. But Windows 11 will be released very soon, and I want to update the application interface and icons, such as in Windows 11.

It seems to me that in a year this will be a fairly popular practice among developers

**Comment by CKGrafico** [comment]
I will add to my todo list but cannot promise to do it soon
